cunumeric/coverage.py

```diff
@@ -69,6 +69,8 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
         runtime.record_api_call(name=name, location=location, implemented=True)
         return func(*args, **kwargs)
 
+    wrapper._implemented = True
```

Comment:
This is purely to make testing easier. It could hypothetically cause a conflict, if any of the things we were wrapping also defined the same attribute, but that is not the case
```
In [65]: any(hasattr(x, '_implemented') for x in np.__dict__.values())
Out[65]: False

In [66]: any(hasattr(x, '_implemented') for x in np.ndarray.__dict__.values())
Out[66]: False

In [67]: any(hasattr(x, '_implemented') for x in np.linalg.__dict__.values())
Out[67]: False
```